asreview/webapp/src/StyledComponents/StyledAlertPrior.js

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { Alert, Box } from "@mui/material";
+
+export function ExplorationModeRecordAlertPrior(props) {
```

Comment:
What's the difference with StyledAlert?

Reply:
`StyledAlertPrior` is used in the prior selection screen and displays a short text:
```
Initially labeled as [relevant/irrelevant/not_seen]
```

and `StyledAlert` is used on the reviewing screen and displays a message:
```
Initially labeled as [relevant/irrelevant/not_seen], what would be your decision?
```

Reply:
Oke, let's merge the two classes and add argument `prior`. 

```{props.prior : "" ? ", what would be your decision?"}

Reply:
Implemented